### DISCUSSION TITLE
Are backups compressed?

### DISCUSSION THREAD

**Discussion by petervk** [lavinir/hassio-onedrive-backup#53] [General]
Hi! When I look in the hassio-onedrive-backup folder in my onedrive there are a bunch of *.tar files. Does this mean the backups are uncompressed? I might not fully understand how this works but my assumption was that the files would be *.tar.gz if they were gzip compressed. If they aren't compressed, is this something that can/should be added?

**Comment by lavinir** [comment]
All backups are compressed by Home Assistant. If you open the TAR file you'll see the individual components backed up are each inside a tar.gz file and are all just grouped in a TAR file (the backup file).

**Reply by petervk** [reply]
Thank you!
